js/client/modules/@arangodb/testutils/testing.js

```diff
@@ -553,6 +564,27 @@ function iterateTests(cases, options) {
   return results;
 }
 
+function getFailedTestCases(options) {
```

Comment:
please move this function to `js/client/modules/\@arangodb/testutils/result-processing.js`